src/codeu/chat/server/JSON.java

```diff
@@ -0,0 +1,110 @@
+package codeu.chat.server;
+
+import java.io.File;
+import java.io.IOException;
+
+import codeu.chat.common.ConversationHeader;
+import codeu.chat.common.Message;
+import codeu.chat.common.User;
+
+import com.fasterxml.jackson.core.JsonGenerationException;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.util.DefaultPrettyPrinter;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.ObjectWriter;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+
+public final class JSON {
+
+	private static void createNewJSON(ObjectMapper mapper) {
+		ObjectNode outerNode = mapper.createObjectNode();
+		outerNode.set("users", mapper.createArrayNode());
+		outerNode.set("conversations", mapper.createArrayNode());
+		outerNode.set("messages", mapper.createArrayNode());
+		ObjectWriter writer = mapper.writer(new DefaultPrettyPrinter());
+		try {
+			writer.writeValue(new File("data.json"), outerNode);
```

Comment:
You probably want to take this as a parameter, so you're putting it in the data directory specified by the user? This will become useful if you start using Google's hosting.